### PR TITLE
Support pluralization

### PR DESCRIPTION
**Pull request by Silex**
:tada:

### CODE REVIEW

**Comment by Silex** [comment]
Actually we don't even need to call `pluralize` in the backend because it's already done. It's just the frontend helper that needs to use the correct key.

Maybe that's because if `rails-18n` tho, I'll check.

**Comment by Silex** [comment]
Okay, basic pluralization is already present in i18n by default (https://github.com/svenfuchs/i18n/blob/master/lib/i18n/backend/base.rb#L48).

That means we are good to go!

If we wanted to support different pluralisation rules, we'd need to include https://github.com/svenfuchs/i18n/blob/master/lib/i18n/backend/pluralization.rb (see https://github.com/svenfuchs/i18n/wiki/Pluralizations) but that'd be complicated I think.

**Comment by Silex** [comment]
The editing does not work if the translation does not already exist... not sure how to fix this.

**Comment by Silex** [comment]
Ok I fixed it by moving the pluralization one line up, that way the backend stores it. This can be merged.

File: app/helpers/lit/frontend_helper.rb [hunk 1]
```diff
@@ -15,6 +16,17 @@ def translate(key, options = {})
       def t(key, options = {})
         translate(key, options)
       end
+
+      def pluralize_key(key, count)
```
Comment (mlitwiniuk) [inline app/helpers/lit/frontend_helper.rb]: it's not that easy - pluralization rules are different for different languages, ie. please take a look here: https://github.com/svenfuchs/rails-i18n/blob/4223a0010e81242c69dfb5cde7b314f79cea2f0b/rails/pluralization/ar.rb

So this code should take under consideration different rules depending on language code.

Comment (Silex) [reply app/helpers/lit/frontend_helper.rb]: Okay, basically I just need to fix `pluralize_key` :wink:

**Comment by Silex** [comment]
Okay this seems to work well.

**Comment by Silex** [comment]
:+1:

**Pull request merged**